mollie/api/objects/capture.py

```diff
@@ -0,0 +1,40 @@
+from .base import Base
+
+
+class Capture(Base):
+    @classmethod
+    def get_resource_class(cls, client):
+        from ..resources.captures import Captures
+        return Captures(client)
+
+    @property
+    def id(self):
+        return self._get_property('id')
+
+    @property
+    def mode(self):
+        return self._get_property('mode')
+
+    @property
+    def amount(self):
+        return self._get_property('amount')
+
+    @property
+    def settlement_amount(self):
+        return self._get_property('settlementAmount')
+
+    @property
+    def payment_id(self):
+        return self._get_property('paymentId')
+
+    @property
+    def shipment_id(self):
+        return self._get_property('shipmentId')
```

Comment:
This property is not under test, see https://travis-ci.org/mollie/mollie-api-python/jobs/535273819

Reply:
Additionally, we should also have the property `shipment` (since that is defined in the `_links`). Same for `payment` and `settlement`. See f.i: https://github.com/mollie/mollie-api-python/blob/e6c66b8283d852536a54bfc38028febf6bc4f82a/mollie/api/objects/refund.py#L69

Reply:
You didn't add any test for the added properties... Use the coverage output, Luke ;)

Reply:
This is still open

Reply:
what do you mean?